examples/event_handling/poly_editor.py

```diff
@@ -141,6 +141,10 @@ def motion_notify_callback(self, event):
         x, y = event.xdata, event.ydata
 
         self.poly.xy[self._ind] = x, y
+        if self._ind == 0:
+            self.poly.xy[-1] = x, y
+        elif self._ind == len(self.poly.xy) - 1:
```

Comment:
It's [not possible](https://github.com/matplotlib/matplotlib/blob/master/examples/event_handling/poly_editor.py#L74).